lib/daru/io/importers/redis.rb

```diff
@@ -0,0 +1,176 @@
+require 'daru'
+
+require 'daru/io/importers/util'
+require 'json'
+
+module Daru
+  module IO
+    module Importers
+      class Redis
+        # Imports a *Daru::DataFrame* from *Redis* connection and keys.
+        #
+        # @note In Redis, the specified key and count the number of queries that
+        #   do not always fit perfectly. This persists in this module too,
+        #   as this module is built on top of redis Ruby gem. Hence, if a query
+        #   for 100 keys doesn't return exactly 100 keys, it is not a bug in
+        #   this module. It is just how Redis works.
+        #
+        # @param connection [Hash or Redis Instance] Either a Hash of *Redis* configurations,
+        #   or an existing *Redis* instance. For the hash configurations, have a
+        #   look at {http://www.rubydoc.info/github/redis/redis-rb/Redis:initialize
+        #   Redis#initialize}.
+        # @param keys [Array] Redis key(s) from whom, the *Daru::DataFrame*
+        #   should be constructed. If no keys are given, all keys in the *Redis*
+        #   connection will be used.
+        # @param match [String] A pattern to get matching keys.
+        # @param count [Integer] Number of matching keys to be obtained.
+        # @param page [Integer] The page of keys to be fetched from the
+        #   paginated results of matching keys. Defaults to 0 (ie, First
+        #   page results). For the i-th page results, pass the argument
+        #   as i-1. When passed as a negative number, the whole pagination
+        #   is traversed and ALL matching keys are used.
```

Comment:
TBH, I don't believe that Redis's "pages" have any value for daru user. I'd say probably we can have `limit` and `offset` to have "entries from 1001 to 1100", but just `limit` (or `count`) will work for initial implementation.